packages/render-ui/src/parseHtmlHasText.js

```diff
@@ -0,0 +1,29 @@
+export const parseHtmlHasText = input => {
```

Comment:
rename please - see above ..
also I don't see why you are doing things like this  could you not simplify like: 
```javascript
const dp = new DOMParser();

let markupToText = (s) => { 
   const root = dp.parseFromString(s, 'text/html');
   return root.body.textContent;
}

let hasContent = (s) => {
  if(!s){
    return false;
  }
  let tc = markupToText(s);
  return tc && tc.trim() ? true : false;
}
console.log('1', true, hasContent('<div>Rationale</div>'));
console.log(2, true, hasContent('Rationale'));
console.log(3, false, hasContent('<div></div>'));
console.log(4, false, hasContent('<div> </div>'));
console.log(5, false, hasContent(' '));
console.log(6, false, hasContent(null));
```

?

Reply:
sure, I'll simplify the function